pkg/apiserver/rest/usecase/workflow.go

```diff
@@ -268,11 +287,80 @@ func (w *workflowUsecaseImpl) DetailWorkflowRecord(ctx context.Context, workflow
 	}, nil
 }
 
-func (w *workflowUsecaseImpl) createWorkflowRecord(ctx context.Context, revision *appsv1.ControllerRevision) error {
-	app, err := util.RawExtension2Application(revision.Data)
+func (w *workflowUsecaseImpl) SyncWorkflowRecord(ctx context.Context) error {
+	crList := &appsv1.ControllerRevisionList{}
```

Comment:
What specifically is stored in ControllerRevision? model.DeployEvent?

Reply:
The whole application data is in the controller revision.